docs/rfcs/014-born_digital_workflow/README.md

```diff
@@ -0,0 +1,176 @@
+
+- Appraisal and selection will remain manual until the functionality in Archivematica improves
+- Cataloguing will be done in Calm and file-level description will not be part of the catalogue record
+- All processing steps within Archivematica will be fully automated and require no intervention
+
+### Overview
+
+![Workflow diagram](workflow.png)
+
+The workflow consists of the following high-level stages:
+
+1. Appraisal
+2. Cataloguing
+3. Transfer
+4. Processing
+5. Storage
+
+Each of these are described below.
+
+### Appraisal
+
+During the appraisal stage, an archivist will determine which files are in scope for the collections and perform a policy-led sensitivity check. We intend to move towards system-assisted appraisal, using future version of Archivematica, but for now this stage will build on existing manual processes.
+
+| Step                 | Description |
+|----------------------|-------------|
+| Disk level appraisal | High level appraisal to determine what storage media is in scope. |
+| File level appraisal | Selection of in scope files from selected storage media. |
+| Sensitivity check    | Policy-led review of selected files to determine whether they contain sensitive material. |
+
+### Cataloguing
+
+Once an initial appraisal has been completed, the files in scope for archiving need to be arranged into archive items and catalogued in Calm. Items should be created in Calm to represent each related set of files, which will be submitted and stored as a single package in the storage service.
+
+This means that an item in Calm corresponds to a bag in the storage service, not an individual file. File-level description is fully automated and the metadata generated is stored in a METS file created by the Archivematica processing workflow.
+
+| Step                   | Description |
+|------------------------|-------------|
+| Find/create collection | Find, or create if necessary, the collection in Calm that the items will be part of. |
+| Find/create series     | Find, or create if necessary, the series and sub-series within the collection that the items will be part of. |
+| Create items           | For each set of related files, create an item in Calm that represents and describes them collectively in the catalogue. This should describe the files as a set, not list details of individual files. |
+
+### Transfer
+
+Once each set of related files as been identified, they need to be assembled into a package that can be transferred into Archivematica for processing and storage. This package is a simple zip file, with the addition of a minimal set of descriptive metadata.
+
+This metadata is used for identification only. The canonical version of the metadata, which is also the data that will be displayed to end users, remains the data in Calm. Similarly, whilst Archivematica does have support for rights information, we will not be supplying this through the workflow. Calm will be the sole location for rights information.
```

Comment:
To clarify, does this mean that the metadata in Calm will display in the Universal Viewer rather than the basic metadata packaged with the images in much the same way that books/archival material digitzed through Goobi has some metadata with it but ultimately the UV is actually displaying most of the metadata from Sierra? So, as changes about an item are made in Sierra, the UV will display the most correct metadata? And in the instance of born digital, will the UV just be displaying the information from CALM itself or from a record in Sierra harvested from CALM?

Reply:
I definitely want to avoid dragging anything through Sierra. The sync to Calm is one way only and is a massive headache for my team. 

This may - almost certainly will - result new workflows / code being required to enable us to render born digital items online. But we need to think that through anyway vis-a-vis sensitivity.

Reply:
Indeed, this workflow won't result in anything being made available online at this point in time.

To do that will require us to have the Calm data in the Catalogue API, support for born-digital on the new site and the ability to securely authenticate and delivery assets that we can make available.

So this won't result in things going into the UV or require any metadata in Sierra.